### PR TITLE
[Communication] - azure-communication-phonenumbers - Update default polling interval value

### PR DESCRIPTION
**Pull request by sorrego-msft**
# Description

This will set a default polling interval of 2 seconds for the functions: begin_search_available_phone_numbers, begin_purchase_phone_numbers and begin_update_phone_number_capabilities.

### CODE REVIEW

File: sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_phone_numbers_client.py [hunk 1]
```diff
@@ -95,8 +95,10 @@ def begin_purchase_phone_numbers(
             for LRO operations if no Retry-After header is present.
```
Comment (automagically) [inline sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_phone_numbers_client.py:95]: Would be nice to take the time to document what is the unit of the polling interval e.g. seconds

File: sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_phone_numbers_client.py [hunk 2]
```diff
@@ -95,8 +95,10 @@ def begin_purchase_phone_numbers(
...
         :rtype: ~azure.core.polling.LROPoller[None]
         """
+        polling_interval = kwargs.pop('polling_interval', 2)
```
Comment (automagically) [inline sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_phone_numbers_client.py:98]: First of all, kudos for using the default value for the pop function, second this '2' is a magic number that would be more readable within a constant like `_DEFAULT_POLLING_INTERVAL_IN_SECONDS = 2` of course change seconds by the real unit of the pooling interval.

**Review by automagically — CHANGES_REQUESTED** [review]

**Comment by sorrego-msft** [comment]
> @sorrego-msft please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

**Review by automagically — APPROVED** [review]

**Review by danielortega-msft — CHANGES_REQUESTED** [review]
The change looks good, but I believe you would need to replicate it to the async client as well (sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/aio/_phone_numbers_client_async.py)

**Review by danielortega-msft — APPROVED** [review]

**Pull request merged**